dm/dm/worker/subtask.go

```diff
@@ -151,7 +150,7 @@ func (st *SubTask) initUnits(relay relay.Process) error {
 	// NOTE: because lightning not support init tls with raw certs bytes, we write the certs data to a file.
 	if st.cfg.NeedUseLightning() && st.cfg.To.Security != nil {
 		// NOTE: LoaderConfig.Dir is always not empty because we only dump certs when we use lightning.
-		if err := st.cfg.To.Security.DumpTLSContent(filepath.Join(st.cfg.LoaderConfig.Dir, "..")); err != nil {
+		if err := st.cfg.To.Security.DumpTLSContent("./" + loader.TmpTLSConfigPath + "_" + st.cfg.Name); err != nil {
```

Comment:
In the old PR I'm concerned about the certificate files are leaked to other machines by dumping them. Not sure what's our opinions now.

Reply:
The reason we need to fix is `LoaderConfig.Dir` may be s3 path, but we don't implement tls at s3 in lightning, so we need to dump it to a local path but not `LoaderConfig.Dir`.

Reply:
I mean if we only keep the certificate in memory, there's less risk to leak it. It's an old problem but may be a security issue @sunzhaoyang

Reply:
btw i forget why not just dump certs to `st.cfg.LoaderConfig.Dir`  😵😵

Reply:
this is because `st.cfg.LoaderConfig.Dir` may be a path in s3

Reply:
i add some notes to reduce ambiguity.